src/util/specific.py

```diff
@@ -25,9 +25,22 @@ def dbuuid(cr):
     return cr.fetchone()[0]
 
 
-def dispatch_by_dbuuid(cr, version, callbacks):
+@_cached
+def dbuuids(cr):
```

Comment:
Let's make this "private".
```suggestion
def _dbuuids(cr):
```

Reply:
Somewhat relevant discussion: should we deprecate the former `dbuuid()` util? Reason being that it will prioritise the origin dbuuid, when available.

Reply:
I think we don't need to worry much about that for the issue at hand. Which is more related to the dispatcher.

Reply:
It's not retro-compatible, then.